clients/src/main/java/org/apache/kafka/common/TopicPartitionInfo.java

```diff
@@ -79,9 +95,24 @@ public List<Node> isr() {
         return isr;
     }
 
+    /**
+     * Return the eligible leader replicas of the partition. Note that the ordering of the result is unspecified.
+     */
+    public List<Node> eligibleLeaderReplicas() {
+        return elr;
+    }
+
+    /**
+     * Return the last known eligible leader replicas of the partition. Note that the ordering of the result is unspecified.
+     */
+    public List<Node> lastKnownEligibleLeaderReplicas() {
+        return lastKnownElr;
+    }
+
     public String toString() {
         return "(partition=" + partition + ", leader=" + leader + ", replicas=" +
-            Utils.join(replicas, ", ") + ", isr=" + Utils.join(isr, ", ") + ")";
+            Utils.join(replicas, ", ") + ", isr=" + Utils.join(isr, ", ") +
+            ", elr=" +Utils.join(elr, ", ") + ", lastKnownElr=" + Utils.join(lastKnownElr, ", ") + ")";
```

Comment:
Just the nitiest of picks...

```suggestion
            ", elr=" + Utils.join(elr, ", ") + ", lastKnownElr=" + Utils.join(lastKnownElr, ", ") + ")";
```